src/os_esp32.cc

```diff
@@ -601,6 +604,9 @@ class HeapSummaryPage {
     return "unknown";
   }
 
+  Process* get_toit_process() const { return owning_process_; }
```

Comment:
I'd try to be more consistent with the naming. Not sure the toit_ prefix helps much here. Maybe just call this `owning_process`?